docs/docs/en/guide/installation/pseudo-cluster.md

```diff
@@ -83,6 +83,7 @@ each machine. You could find this file in the path `bin/env/install_env.sh` and
 # ---------------------------------------------------------
 # Due to the master, worker, and API server being deployed on a single node, the IP of the server is the machine IP or localhost
 ips="localhost"
+sshPort=${sshPort:-"22"}
```

Comment:
how about we directly using `sshPort="22"`, and we add some sentence at the end of `Modify Configuration` section tell users they can change the configuration by environment, like `export sshPort="22"`

Reply:
Add I think we should better separate those change in other PR